Sources/TuistLoader/SwiftPackageManager/PackageInfoMapper.swift

```diff
@@ -795,9 +795,14 @@ extension ProjectDescription.ResourceFileElements {
         /// - Parameters:
         ///   - resourceAbsolutePath: The absolute path of that resource
         /// - Returns: A ProjectDescription.ResourceFileElement mapped from a `.process` resource rule of SPM
-        func handleProcessResource(resourceAbsolutePath: AbsolutePath) throws -> ProjectDescription.ResourceFileElement {
+        func handleProcessResource(resourceAbsolutePath: AbsolutePath) throws -> ProjectDescription.ResourceFileElement? {
```

Comment:
Is the issue only happening with process resources?

Reply:
Based on my research, it happens only when `sources` is missing in package manifest. In this case, Tuist uses `process`

https://github.com/tuist/tuist/blob/1fc08abdb3905a2f330d8df8942866a504727899/Sources/TuistLoader/SwiftPackageManager/PackageInfoMapper.swift#L841

Reply:
I was able to fix warnings by this PR for these repos:

https://github.com/SVProgressHUD/SVProgressHUD
https://github.com/kasketis/netfox

I could try to find a repo with existing `sources`, `resources` with `copy`, and `exclude` with glob, but it's not easy to find.

Reply:
> I could try to find a repo with existing sources, resources with copy, and exclude with glob, but it's not easy to find.

I think those are sufficient. We can address the others as they arise, there's no need to go and find them.
Thanks a lot @kapitoshka438